extensions/sd_api_pictures/script.py

```diff
@@ -265,6 +268,23 @@ def custom_css():
     path_to_css = Path(__file__).parent.resolve() / 'style.css'
     return open(path_to_css, 'r').read()
 
+def get_checkpoints():
+    global params
+
+    models = requests.get(url=f'{params["address"]}/sdapi/v1/sd-models')
+    options = requests.get(url=f'{params["address"]}/sdapi/v1/options')
+    options_json = options.json()
+    params['sd_checkpoint'] = options_json['sd_model_checkpoint']
+    params['checkpoint_list'] = [result["title"] for result in models.json()]
+    return gr.update(choices=params['checkpoint_list'], value=params['sd_checkpoint'])
```

Comment:
```suggestion
    try:
        models = requests.get(url=f'{params["address"]}/sdapi/v1/sd-models')
        options = requests.get(url=f'{params["address"]}/sdapi/v1/options')
        options_json = options.json()
        params['sd_checkpoint'] = options_json['sd_model_checkpoint']
        params['checkpoint_list'] = [result["title"] for result in models.json()]
    except:
        params['sd_checkpoint'] = ""
        params['checkpoint_list'] = []

    return gr.update(choices=params['checkpoint_list'], value=params['sd_checkpoint'])
```
This way it fails gracefully.